Terminal.Gui/Views/ListView.cs

```diff
@@ -736,14 +736,23 @@ public override bool OnEnter (View view)
 		/// </summary>
 		public void EnsureSelectedItemVisible ()
 		{
-			SuperView?.LayoutSubviews ();
-			if (selected < top) {
-				top = Math.Max (selected, 0);
-			} else if (Frame.Height > 0 && selected >= top + Frame.Height) {
-				top = Math.Max (selected - Frame.Height + 1, 0);
+			if (SuperView?.IsInitialized == true) {
```

Comment:
if (SuperView?.IsInitialized == true) -> if (SuperView?.IsInitialized)

Reply:
It using a nullable so it is needed explicit the true.